setup.py

```diff
@@ -16,6 +17,7 @@
             'pytest',
             'pylint',
             'pytest-timeout',
+            'pyserial',
```

Comment:
This isn't a dev requirement, this is a standard requirement, so this should go into `install_requires`

Reply:
Resolved with 662a21c